styles.css

```diff
@@ -83,7 +84,155 @@
   font-weight: 400;
 }
 
-#horizontal-line {
-  border: 0.5px solid #646369;
+.horizontal-line {
+  border: 0.2px solid #646369;
   margin: 25px;
 }
+
+.about-section {
+  display: grid;
+  min-height: 40vh;
+  grid-template-columns: auto auto;
+  padding: 1rem;
+  gap: 10px;
+}
+
+.title {
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  color: #9490ff;
+  padding: 0.6rem 1.2rem 0.6rem 1.2rem;
+}
+
+.skills {
+  list-style-type: none;
+  padding: 1.2rem;
+  color: white;
+  font-size: 15px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 24px;
+}
+
+.skills > li {
+  margin-bottom: 0.3rem;
+}
+
+.social-icons {
+  min-height: 10vh;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.social-icons > a {
+  color: white;
+  padding: 5px;
+  zoom: 1.5;
+  text-decoration: none;
+}
+
+.work-section {
+  display: grid;
+  min-height: 50vh;
+}
+
+.work-section-title-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  color: #a2a3ae;
+}
+
+.work-section-title {
+  text-align: center;
+  color: white;
+  padding: 10px;
+  font-size: 20px;
+  font-style: normal;
+  line-height: 32px;
+  font-weight: 500;
+}
+
+.work-icon {
+  zoom: 2;
+}
+
+.recent-work {
+  color: #9490ff;
+  font-size: 50px;
+  line-height: 60px;
+  font-style: normal;
+  font-weight: 500;
+  padding: 60px 0 60px 1.7rem;
+}
+
+.work-section-hr {
+  margin-top: -60px;
+}
+
+.project {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start). 

- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.Flex is just an example you can do it to duplicate properties.



- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)